File: app/src/main/java/com/bmn/bookfinder/models/googlebooks/ResponsePdf.java
package com.bmn.bookfinder.models.googlebooks;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ResponsePdf {

    private final static long serialVersionUID = 3047883525262745870L;
    @SerializedName("isAvailable")
    @Expose
    private boolean isAvailable;
    @SerializedName("downloadLink")
    @Expose
    private String downloadLink;
    @SerializedName("acsTokenLink")
    @Expose
    private String acsTokenLink;

    public boolean isIsAvailable() {
        return isAvailable;
    }

    public void setIsAvailable(boolean isAvailable) {
        this.isAvailable = isAvailable;
    }

    public String getDownloadLink() {
        return downloadLink;
    }

    public void setDownloadLink(String downloadLink) {
        this.downloadLink = downloadLink;
    }

    public String getAcsTokenLink() {
        return acsTokenLink;
    }

    public void setAcsTokenLink(String acsTokenLink) {
        this.acsTokenLink = acsTokenLink;
    }

}

File: app/src/main/java/com/bmn/bookfinder/models/googlebooks/ResponsePdf.kt
package com.bmn.bookfinder.models.googlebooks

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ResponsePdf {
    @SerializedName("isAvailable")
    @Expose
    var isIsAvailable = false
        private set

    @SerializedName("downloadLink")
    @Expose
    var downloadLink: String? = null

    @SerializedName("acsTokenLink")
    @Expose
    var acsTokenLink: String? = null
    fun setIsAvailable(isAvailable: Boolean) {
        isIsAvailable = isAvailable
    }

    companion object {
        private const val serialVersionUID = 3047883525262745870L
    }
}